### ISSUE TITLE
execv error: No such file or directory

### ISSUE THREAD

**Issue by kholisrag** [flameshot-org/flameshot#2764]
### Flameshot Version

Flameshot-12.1.0.x86_64

### Installation Type

AppImage

### Operating System type and version

Ubuntu 20.04.4 LTS

### Description

flameshot 12.1.0 logs when trying to run from terminal

```bash
❯ ./Flameshot-12.1.0.x86_64_10ab227368a704f2d66cded7a7aa8248.AppImage 
execv error: No such file or directory

```

### Steps to reproduce

1. run appimage from terminal for more verbose logs
2. flameshot didn't run at all

### Screenshots or screen recordings

![Screenshot_2022-07-06_04-26-32](https://user-images.githubusercontent.com/4683704/177419347-141bdb8d-64f8-4a00-a00d-85a91dc66023.png)

### System Information

```
         Icon name: computer-laptop
           Chassis: laptop
  Operating System: Ubuntu 20.04.4 LTS
            Kernel: Linux 5.13.0-52-generic
      Architecture: x86-64

```

**Comment by borgmanJeremy** [comment]
I cannot replicate this is in a clean VM. Do you use a non english locale by chance?

**Comment by kholisrag** [comment]
>  Do you use a non english locale by chance?

```
❯ locale
LANG=en_US.UTF-8
LANGUAGE=
LC_CTYPE="en_US.UTF-8"
LC_NUMERIC=id_ID.UTF-8
LC_TIME=id_ID.UTF-8
LC_COLLATE="en_US.UTF-8"
LC_MONETARY=id_ID.UTF-8
LC_MESSAGES="en_US.UTF-8"
LC_PAPER=id_ID.UTF-8
LC_NAME=id_ID.UTF-8
LC_ADDRESS=id_ID.UTF-8
LC_TELEPHONE=id_ID.UTF-8
LC_MEASUREMENT=id_ID.UTF-8
LC_IDENTIFICATION=id_ID.UTF-8
LC_ALL=

```

> I cannot replicate this is in a clean VM.

any other things I should check, rather than locale? because in flameshot 11, its run, and can be used...

**Comment by borgmanJeremy** [comment]
Did you double check the download did not get corrupted my confirming the sha256?

**Comment by kholisrag** [comment]
seem correct
![Screenshot_2022-07-06_18-45-10](https://user-images.githubusercontent.com/4683704/177542902-167bc54d-00bf-455b-bc39-d25534c7b66a.png)

**Comment by borgmanJeremy** [comment]
hmm, no I am out of ideas then since it works fine in a clean VM.

You could use the flatpak or snap.

**Comment by mmahmoudian** [comment]
@kholisrag Do you have any AppImage helper software? something like the [AppImageLauncher](https://github.com/TheAssassin/AppImageLauncher)?

**Comment by mmahmoudian** [comment]
I cannot reproduce it in a clean VM either:

![image](https://user-images.githubusercontent.com/390889/177980264-4ef0fb3d-59b8-4df1-b724-bff7aefd6a2d.png)

**Comment by kholisrag** [comment]
> @kholisrag Do you have any AppImage helper software? something like the [AppImageLauncher](https://github.com/TheAssassin/AppImageLauncher)?

Yes it is

**Comment by mmahmoudian** [comment]
> > @kholisrag Do you have any AppImage helper software? something like the [AppImageLauncher](https://github.com/TheAssassin/AppImageLauncher)?
> 
> Yes it is

Hmmm, then considering that this is not reproducible in a clean VM, I suggest you try another AppImage and see if the issue is there, and ultimately file a bug report.

I personally don't see a point in using an AppImage helper as all a user needs to do is:

```sh
mkdir -p ~/Applications/flameshot
cd ~/Applications/flameshot
rm -f Flameshot*
wget <the_AppImage_URL>
chmod +x Flameshot*
./Flameshot*
```

I will soon put together a shell script to automate the whole process and even update the AppImage automatically.

**Comment by kholisrag** [comment]
> > > @kholisrag Do you have any AppImage helper software? something like the [AppImageLauncher](https://github.com/TheAssassin/AppImageLauncher)?
> 
> > 
> 
> > Yes it is
> 
> 
> 
> Hmmm, then considering that this is not reproducible in a clean VM, I suggest you try another AppImage and see if the issue is there, and ultimately file a bug report.
> 
> 
> 
> I personally don't see a point in using an AppImage helper as all a user needs to do is:
> 
> 
> 
> ```sh
> 
> mkdir -p ~/Applications/flameshot      # create a folder to store the AppImage if it does not exist
> 
> cd ~/Applications/flameshot            # navigate to the folder
> 
> rm -f Flameshot*.AppImage              # remove previous AppImages if any exists
> 
> wget <the_AppImage_URL>                # download the new one (remember to provide the URL)
> 
> chmod +x Flameshot*.AppImage           # make it executible
> 
> ./Flameshot*.AppImage                  # execute it (from now on you only need this line only)
> 
> ```
> 
> 
> 
> I will soon put together a shell script to automate the whole process and even update the AppImage automatically.

When using flameshot 11 and bitwarden appimage, all work well

**Comment by mmahmoudian** [comment]
@kholisrag is flameshot the only AppImage that has this issue? If so, please provide version number of the AppImageLauncher and how you have installed it so that I can try to replicate the setup in a VM and investigate further

**Comment by hsequeda** [comment]
I've the same error with Flameshot 12.1.0, using the AppImage in Manjaro with BSPWM.

**Comment by mmahmoudian** [comment]
@hsequeda you are using AppImage Launcher which unfortunately by default is installed on Manjaro. This is not a Flameshot issue, but rather the AppImageLauncher issue. Report to them or alternatively, delete it from your system and reboot. I have reported this Manjaro many times, and the last time I have reported this was 9 months ago:

https://forum.manjaro.org/t/trouble-running-appimages-after-removing-appimagelauncher/100297

No one need an extra software to run AppImage. You just need to run it as a normal executable file.
